### DISCUSSION TITLE
Footnotes?

### DISCUSSION THREAD

**Discussion by RenaudDenis** [nunocoracao/blowfish#1755] [Q&A]
Is there any way to enable Markdown footnotes, as described in the [extended syntax guide](https://www.markdownguide.org/extended-syntax/#footnotes)?

**Comment by jamesbraza** [comment]
Fwiw footnotes seem to work for me, with Hugo version 0.148.0 and Blowfish version 2.88.1:

```md
Hello [^1]

[^1]: world
```

<img width="346" height="188" alt="screenshot of footnotes" src="https://github.com/user-attachments/assets/2c058cd4-a2ce-4fb7-b94f-8a9899724583" />
